README.md

```diff
@@ -25,7 +25,7 @@ This README file only contains basic setup instructions.
 Spark is built using [Apache Maven](https://maven.apache.org/).
 To build Spark and its example programs, run:
 
-    build/mvn -DskipTests clean package
+    ./build/mvn -DskipTests clean package
```

Comment:
Could you find more instances, @Mister-Meeseeks ? For example, `R/README.md` also has this.